Do test on Python 3.7

Freso opened the pull request:
Picard supports Python 3.7, so making sure plugins work on it is very useful.

Heavily inspired by https://github.com/metabrainz/picard-plugins/pull/177 ;)

Freso committed:
Do test on Python 3.7

Picard supports Python 3.7, so making sure plugins work on it is
very useful.

Heavily inspired by
https://github.com/metabrainz/picard-plugins/pull/177 ;)

Freso commented:
Ugh. Looks like the Travisbuntu is too ancient to have Py3.7? I'll try and look into this later.

phw commented:
Unfortunately it won't work that easily, unless they have Python 3.7 finally available on Travis CI :) See https://github.com/metabrainz/picard/commit/c0448cbbf983d7af64421843eecc78d383633f03 for a working solution if the builds will fail

Freso committed:
Use Ubuntu 16.04 Xenial test environment distro

Ubuntu 14.04 Trusty (Travis' default distribution) doesn't contain
Python 3.7 and is generally more than 4 years old and I think not
supported by Picard 2.0 anyway, so moving to a 2 years never version of
Ubuntu that does contain Python 3.7.
